salt/covid19public.sls

```diff
@@ -0,0 +1,40 @@
+{% from 'lib.sls' import apache, create_user %}
+
+
+# So far, all servers with Python apps use Apache and uWSGI. If we later have a server that doesn't need these, we can
+# add boolean key to the Pillar data to indicate whether to include these.
+include:
+  - apache
+  - apache.modules.proxy_http
+  - apache.modules.proxy_uwsgi
```

Comment:
These modules aren't used by `react-js.conf.include`

```suggestion
```

Reply:
removed these proxy http and uwsgi